case-lib/config.sh

```diff
@@ -90,3 +90,13 @@ FALLBACK_TO_PROC=${FALLBACK_TO_PROC:-false}
 
 # Skip the storage checks if they aren't required
 ENABLE_STORAGE_CHECKS=${ENABLE_STORAGE_CHECKS:-0}
+
+# start_test in lib.sh will check whether FW is loaded before every test case.
+# If FW LOADING is not a prerequisite for the test, or if you just want to skip
+# the check, then define NO_POLL_FW_LOADING to true.
+# NO_POLL_FW_LOADING=true
+
+# Set MAX Polling time to check FW Loading. If FW is already loaded, it will
+# return immediately. Default value is set to 60 seconds, because i915 driver
+# timeout is 60 seconds.
+MAX_WAIT_FW_LOADING=${MAX_WAIT_FW_LOADING:-60}
```

Comment:
what is the relationship between DSP firmware load and i915? is this that we first request the i915 modules before downloading the firmware?

and if yes, should we use the same routine as for the kmod remove/modprobe test where we have an iteration that stops on BOOT_COMPLETE.

Reply:
> and if yes, should we use the same routine as for the kmod remove/modprobe test where we have an iteration that stops on BOOT_COMPLETE.

That's exactly the suggestion I made in internal bug 387 and what @fredoh9 is doing here: he's re-using the functions I wrote in #862